zarr/tests/test_core.py

```diff
@@ -1623,16 +1623,13 @@ def test_nbytes_stored(self):
 
 
 @skip_test_env_var("ZARR_TEST_ABS")
+@pytest.mark.usefixtures("azurite")
 class TestArrayWithABSStore(TestArray):
 
     @staticmethod
     def absstore():
-        asb = pytest.importorskip("azure.storage.blob")
```

Comment:
There's a slight change here: previously, setting `ZARR_TEST_ABS` but not having the dependencies would silently pass thanks to the `importorskip`. Now if you set `ZARR_TEST_ABS` we assume you want those tests to run, and fail if the dependencies aren't there.